lib/str.js

```diff
@@ -937,22 +937,15 @@ const Str = {
     },
 
     /**
-     * We validate mentions by checking if it's first character is an allowed character
-     * and by checking that we make sure it isn't inside other tags where mentions aren't allowed.
-     * For example, *@username@expensify.com* is a valid mention because we allow bold styling for it,
-     * but `@username@expensify.com` is not because we do not allow mentions within code
+     * We validate mentions by checking if it's first character is an allowed character.
      *
      * @param {String} mention
      * @returns {bool}
      */
     isValidMention(mention) {
         // A valid mention starts with a space, *, _, #, ', ", or @ (with no preceding characters).
         const startsWithValidChar = /[\s*~_#'"@]/g.test(mention.charAt(0));
-
-        // We don't support mention inside code or codefence styling,
-        // for example using `@username@expensify.com` or ```@username@expensify.com``` will be invalid.
-        const containsInvalidTag = /(<code>|<pre>|&#x60;)/g.test(mention);
-        return startsWithValidChar && !containsInvalidTag;
+        return startsWithValidChar;
```

Comment:
You can combine this the above line. There's no added value to having the extra variable

Reply:
You're right. Updated.